photosynthesis_metrics/utils.py

```diff
@@ -24,6 +24,7 @@ def _adjust_dimensions(input_tensors: Union[torch.Tensor, Tuple[torch.Tensor, to
 
 def _validate_input(
         input_tensors: Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]],
+        supports_5d: bool,
```

Comment:
Does "enable_5d" sound better?

Reply:
Sounds better, but I think that `supports_5d` better reflects the fact that the current metric supports 5d tensors

Reply:
As discussed offline, the new naming will be `allow_5d` because:
- Reflects action in contrast to `supports` which is more intuitive for functions
- Shows that we **allow** such type of an input in this particular case